### PR TITLE
Enable CheckStyle Plugin in auth module

### PR DESCRIPTION
**Pull request by liudezhi2098**
Motivation
enable checkStyle_plugin  in  pulsar-broker-auth-athen module.
enable checkStyle_plugin  in pulsar-broker-auth-sasl module.
enable checkStyle_plugin  in pulsar-client-auth-athenz module.
enable checkStyle_plugin  in pulsar-client-auth-sasl module.

Documentation
- [x] `no-need-doc`

### CODE REVIEW

File: pulsar-client-auth-athenz/src/main/java/org/apache/pulsar/client/impl/auth/package-info.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/**
+ * Helper class to access AirCompressor package private classes.
+ */
+package org.apache.pulsar.client.impl.auth;
```
Comment (Shoothzj) [inline pulsar-client-auth-athenz/src/main/java/org/apache/pulsar/client/impl/auth/package-info.java:23]: better to a new line on file end

File: pulsar-client-auth-sasl/src/main/java/org/apache/pulsar/client/impl/auth/package-info.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/**
+ * Helper class to access AirCompressor package private classes.
+ */
+package org.apache.pulsar.client.impl.auth;
```
Comment (Shoothzj) [inline pulsar-client-auth-sasl/src/main/java/org/apache/pulsar/client/impl/auth/package-info.java:23]: better to a new line on file end

**Review by Shoothzj — APPROVED** [review]

**Review by HQebupt — APPROVED** [review]

**Review by mattisonchao — APPROVED** [review]
LGTM +1

**Review by codelipenghui — APPROVED** [review]

**Pull request merged**